.github/workflows/release.yml

```diff
@@ -7,6 +7,13 @@ on:
         description: 'Version'
         required: true
         type: string
+      # debian_revision indicates number of revisions made to the debian configuration without making changes to the source code.
+      # we would set the revision to 2, if we needed to release a new debian package, but the source code of AzureAuth itself had not changed.
+      debian_revision:
+        description: 'Debian revision'
+        required: true
+        default: '1'
+        type: string

```

Comment:
As discussed, unfortunately this can never change without also changing the entire GitHub release version, so I don't think it needs to be a parameter to the release job. We can hardcode this for now.

Reply:
This makes sense.